rpms/base.pan

```diff
@@ -2,16 +2,49 @@ unique template rpms/base;
 
 variable OS_CORE_ONLY ?= false;
 variable OS_CORE_ISCSI_ENABLED ?= false;
+variable OS_CORE_RDMA_ENABLED ?= false;
+
 
 # Standard YUM groups for all machines
-include {'rpms/group/core' };
-include { if ( ! OS_CORE_ONLY ) 'rpms/group/base' };
-include { if ( ! OS_CORE_ONLY ) 'rpms/management-utils' };
+
+'/software/groups' = {
+  # Default packages are mainly firmware for Wifi
+  SELF['core'] = dict('mandatory', true,
+                      'default',false,
+                      'optional', false,
+                     );
```

Comment:
I am aware that there is probably a lot of unconsistency, as we never discussed/described the standard formatting that should be used, and every new contributor tends to have its own preferences... I suggest keeping this for now and starting a discussion on the mailing list. If we converge, adopt the consensus for future PRs and implement reformatting when modifying the existing templates. If we don't converge, put it on the agenda for the next workshop! Would you agree?